lighthouse-core/report/html/renderer/performance-category-renderer.js

```diff
@@ -153,30 +153,30 @@ class PerformanceCategoryRenderer extends CategoryRenderer {
     element.appendChild(metricAuditsEl);
 
     // Opportunities
-    const hintAudits = category.audits
-        .filter(audit => audit.group === 'perf-hint' && audit.result.score < 1)
+    const oppAudits = category.audits
+        .filter(audit => audit.group === 'load-opportunities' && audit.result.score < 1)
         .sort((auditA, auditB) => auditB.result.rawValue - auditA.result.rawValue);
-    if (hintAudits.length) {
-      const maxWaste = Math.max(...hintAudits.map(audit => audit.result.rawValue));
+    if (oppAudits.length) {
+      const maxWaste = Math.max(...oppAudits.map(audit => audit.result.rawValue));
       const scale = Math.ceil(maxWaste / 1000) * 1000;
-      const hintAuditsEl = this.renderAuditGroup(groups['perf-hint'], {expandable: false});
-      hintAudits.forEach(item => hintAuditsEl.appendChild(this._renderPerfHintAudit(item, scale)));
-      hintAuditsEl.open = true;
-      element.appendChild(hintAuditsEl);
+      const oppsEl = this.renderAuditGroup(groups['load-opportunities'], {expandable: false});
+      oppAudits.forEach(item => oppsEl.appendChild(this._renderOpportunity(item, scale)));
+      oppsEl.open = true;
+      element.appendChild(oppsEl);
     }
 
     // Diagnostics
-    const infoAudits = category.audits
-        .filter(audit => audit.group === 'perf-info' && audit.result.score < 1);
-    if (infoAudits.length) {
-      const infoAuditsEl = this.renderAuditGroup(groups['perf-info'], {expandable: false});
-      infoAudits.forEach(item => infoAuditsEl.appendChild(this.renderAudit(item)));
-      infoAuditsEl.open = true;
-      element.appendChild(infoAuditsEl);
+    const diagAudits = category.audits
```

Comment:
does diagnosticAudits push this over line limits? `diag` is a bit more of a stretch IMO :)